test/unit/CMakeLists.txt

```diff
@@ -85,3 +85,20 @@ target_include_directories(boost_url_tests PRIVATE . ../../extra/include)
 target_link_libraries(boost_url_tests PRIVATE Boost::url)
 add_test(NAME boost_url_tests COMMAND boost_url_tests)
 add_dependencies(tests boost_url_tests)
+
+if (CMAKE_CXX_STANDARD GREATER_EQUAL 14)
+    include(FetchContent)
+    FetchContent_Declare(
```

Comment:
We have to decide what to do here if we are testing this. An alternative here is simply not testing the integration and testing the formatter directly.